README.md

```diff
@@ -1,33 +1,51 @@
+# tpm2-software-container
+
 This repository contains metadata and scripts used to generate the container
 images used for continuous integration (CI) by the various tpm2-software
 projects.
 
 ## Local build
 
-To build the container image locally invoke docker with the build command from within the tpm2-software-container directory:
+To build the container image locally invoke [Docker](https://www.docker.com/) or
+[Podman](https://podman.io/) with the build command generated using `make` from
+within the `tpm2-software-container` directory. For example:
 
-```
-docker build .
+```sh
+make
+
+# Build Ubuntu 20.04 image using Docker
+docker build -f ./ubuntu-20.04.docker
+
+# Build Fedora 32 image using Podman, squashing all intermediate layers together
+podman build --squash -f ./fedora-32.docker
 ```
 
 The final output of the build will be an IMAGE ID which can be used to run the container image, for example:
 
-```
+```sh
 docker run -it <IMAGE ID> /bin/bash
 ```
 
-or to run the tpm2-tools CI sequence locally:
+or to run the [tpm2-tools](https://github.com/tpm2-software/tpm2-tools) CI sequence locally:
 
-```
+```sh
 docker run --rm --env-file /path/to/tpm2-tools/.ci/docker.env -v /path/to/tpm2-tools:/workspace/tpm2-tools <IMAGE ID> /bin/bash -c '/workspace/tpm2-tools/.ci/docker.run'
 ```
 
-or to run the tpm2-tss CI sequence locally
+or to run the [tpm2-tss](https://github.com/tpm2-software/tpm2-tss) CI sequence locally
 
-```
+```sh
 docker run --rm --env-file /path/to/tpm2-tss/.ci/docker.env -v /path/to/tpm2-tss:/workspace/tpm2-tss <IMAGE ID> /bin/bash -c '/workspace/tpm2-tss/.ci/docker.run'
 ```
 
 ## Auto builds
 
-The repository is monitored by Github Actions and repository changes are built and submitted to Github Container Registry. The containers are listed on  [tpm2software/packages](https://github.com/orgs/tpm2-software/packages).
+The repository is monitored by Github Actions and repository changes are built
+and submitted to Github Container Registry. The containers are listed on
+[tpm2software/packages](https://github.com/orgs/tpm2-software/packages).
+
+For example, to spawn an environment based on Fedora 32 with Podman:
+
+```sh
+podman run --rm ghcr.io/tpm2-software/fedora-32:latest /bin/bash
```

Comment:
The for example to podman portion seems out of place. We go from saying that you can get the containers here to podman.

Maybe to download and run a container from the registry ....

As always, provide the docker example first :-p